Install.ps1

```diff
@@ -3,7 +3,7 @@
 
 $appDataFolder = GetAppDataFolder
 
-Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
```

Comment:
This was throwing an error when I was trying to run without Administration permission, but this fixes it.